modules/ROOT/pages/amazon/amazon-dynamodb-connector-xml-maven.adoc

```diff
@@ -0,0 +1,75 @@
+= Amazon DynamoDB XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If you manually code a Mule runtime engine (Mule) app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to the connector in your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
```

Comment:
```suggestion
If you manually code a Mule runtime engine (Mule) app in XML, either from the Anypoint Studio XML editor or from a text editor, you can get access to the connector in your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
```

Reply:
Every time I read this, I wonder why someone would want to do this manually instead of using Studio....